### DISCUSSION TITLE
Multiple organizations in profile

### DISCUSSION THREAD

**Discussion by JustinBack** [community/community#70564] [Mobile]
<details>
<summary>If you happen to have multiple Organizations in your profile, clicking on other orgs results in the first one being g opened</summary>
GitHub for iOS, Version 1.135.0 (112648507)
Device: iPhone15,2 (iOS 17.0.3)
</details>

![image](https://github.com/community/community/assets/40178814/291e5309-5b0c-4b40-8485-e876923008f6)

Clicking both orgs results in the first org being opened

**Comment by Vijaykv5** [comment]
@JustinBack if we have multiple org, then clicking on particular orgs will direct you to that org github page right?

I see it works fine :)

**Reply by JustinBack** [reply]
On the Web Client it does, however on mobile whenever I click on @tosdr for example, @jrb-it opens. I can reproduce this on multiple devices.
